com.palantir.typescript/src/com/palantir/typescript/services/language/QuickInfo.java

```diff
@@ -79,6 +82,14 @@ public String getDisplayText() {
         return this.documentation;
     }
 
+    public List<JSDocTagInfo> getTags() {
+        return this.tags;
+    }
+
+    public String getTagsText() {
```

Comment:
Oops... i mean the Hover lol. The lpopup that appear on Hovering. This method would have to be used to fill it with more info.

Sorry.